docs/src/user-manual/glossary.md

```diff
@@ -0,0 +1,368 @@
+goal is generally to avoid technical obsolescence, and target formats are
+typically chosen based on open licenses and specifications, lossless
+compression, ubiquity of support and use, and community consensus on best
+practice. When normalization is performed for access (i.e. during **DIP**
+creation), target formats are typically chosen based on file size and
+compression levels, ease of transmission, and ubiquity of support and adoption
+(particularly on the public web).
+
+## Object
+
+An **intellectual entity** that describes something of enduring value, and under
+consideration for preservation. There is at least one object in every
+**package**, and each object can contain zero or more **files**. An object may
+be described by a **producer** using any desired descriptive **metadata** that
+is meaningful to the designated community.
+
+## Package
+
+An **intellectual entity** describing a type of collection, composed of a set of
+**files** and related **metadata**, assembled together for a particular purpose.
+A package will typically contain one or more **objects**, zero or more
+**files**, and **metadata**. Types of packages used in Enduro include: **SIPs**,
+**PIPs**, **AIPs**, and **DIPs**.
+
+## Pipeline
+
+An instance of a **preservation engine** configured with specific **preservation
+policies**, through which **packages** pass as part of a **workflow**.
+
+## Post-ingest
+
+Post-ingest is a phase in a **preservation workflow** describing all the
+**preservation policy**-defined tasks performed on an **AIP** following
+preservation processing. Post-ingest task examples include: compression and/or
+encryption of AIPs, sending any **metadata** to an external system (such as an
+archival management system), **DIP** generation and delivery, geo-redundant
+**replication**, as well as the transfer of the AIP to its final storage
+location.
+
+## Pre-ingest
+
+Any manual examination, review, and/or additional preparation steps performed on
+**SIPs** received from **producers** prior to **ingest**. Can include activities
+such as appraisal, selection,and SIP arrangement.
+
+## Preservation action
+
+A **workflow** component composed of one or more **tasks** performed on a
+**package** to support preservation, informed by the preservation policies
+configured in the preservation system.
+
+## Preservation engine
+
+The component of Enduro that manages the transformation of **PIPs** into
+**AIPs** in a preservation workflow during preservation processing. See:
+[Components](/src/user-manual/components.md).
+
+## Preservation policy
+
+A policy or business rule defining the target inputs and/or outcomes of a
+preservation action or **workflow** **task**, and any supporting contextual
+information. For example, setting a policy in the **preservation engine** to
+**normalize** a JPEG image file to a JPEG2000 during **AIP** creation.
+
+## Preservation processing
+
+A phase in a preservation
```

Comment:
Short line.

Reply:
Also should "preservation workflow" be bold?

Reply:
lol i missed a lot of terms that should have been bolded in that one. Fixed.